apps/icons/svgrrc.config.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+const path = require('path');
+
+const COPYRIGHT = `
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+`;
+
+/** @type {import('@svgr/core').Config} */
+module.exports = {
+  icon: true,
+  typescript: true,
+  outDir: "./src",
+  jsxRuntime: "automatic",
+  replaceAttrValues: {
```

Comment:
👌🏼

Reply:
```
  svgoConfig: {
    plugins: [{ name: "convertColors", params: { currentColor: true } }],
  },
```
will achieve the same thing, i believe, without having to replace a specific value

Reply:
Sadly I think that'll fail for colors used in clip paths.